src/CMakeLists.txt

```diff
@@ -430,9 +430,6 @@ if(WITH_JAEGER)
     set(jaeger_libs ${CMAKE_BINARY_DIR}/external/lib/libyaml-cpp.so
 		    ${CMAKE_BINARY_DIR}/external/lib/libyaml-cpp.so.0.6
 		    ${CMAKE_BINARY_DIR}/external/lib/libyaml-cpp.so.0.6.2)
-    #customize libjaeger.install
-    execute_process(COMMAND bash -c "sed -i 's/#//' debian/libjaeger.install"
-		    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
```

Comment:
the commit message of

> we will not be using BuildProfileSpec based optional pkg building, removing comment from .install file is no longer needed

is wrong.

Reply:
done: https://github.com/ceph/ceph/pull/38783/commits/d95381c9c581864a2fe7eb56b3288454968bf2bd

Reply:
i'd suggest fold this change into "debian, install-deps.sh: use Build-Profiles to optionally build...".